### PR TITLE
dingonix- added a caption

### PR DESCRIPTION
**Pull request by dingonix**
caption!

### CODE REVIEW

File: _posts/0000-01-03-dingonix.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
```
Comment (beardofedu) [inline _posts/0000-01-03-dingonix.md]: Hey, maybe you should change this title. 

```suggestion
title: "Rainbowtocat to the rescue!!"
```

File: _posts/0000-01-03-dingonix.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
...
 ---
 
-Caption here
+This is a caption.
 
 ![cooler-octocat](https://octodex.github.com/images/twenty-percent-cooler-octocat.png)
```
Comment (beardofedu) [inline _posts/0000-01-03-dingonix.md]: Hey, maybe use a different image from the octodex? https://octodex.github.com

**Review by beardofedu — CHANGES_REQUESTED** [review]
@dingonix just some 💭 s on your changes ...

File: _posts/0000-01-03-dingonix.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: slide
-title: "Welcome to our slide deck!"
+title: "Rainbowtocat to the rescue!!"
```
Comment (grovecj) [inline _posts/0000-01-03-dingonix.md:3]: LGTM

File: _posts/0000-01-03-dingonix.md (hunk 2 above)
Comment (dingonix) [reply _posts/0000-01-03-dingonix.md]: good idea! 😎

**Review by beardofedu — APPROVED** [review]

**Pull request merged**